packages/grafana-ui/.storybook/preview.ts

```diff
@@ -18,7 +18,7 @@ import addons from '@storybook/addons';
 import { ThemedDocsContainer } from '../src/utils/storybook/ThemedDocsContainer';
 
 const handleThemeChange = (theme: any) => {
-  if (theme !== 'light') {
+  if (theme !== 'light' || theme !== 'fusebit') {
     lightTheme.unuse();
```

Comment:
fusebitTheme.unuse();?

Reply:
In practice we don't allow theme changes, so if we can avoid a code change here...

Otherwise, let's properly support it by setting unuse and use consistently across the various cases.

Reply:
I tested if everything worked normally without this change and it does, so i removed it